### ISSUE TITLE
Button in accordion item should be marked aria-disabled when expanded in single select mode

### ISSUE THREAD

**Issue by eljefe223** [microsoft/fast#4160]
**Describe the bug; what happened?**
The button in accordion item should be marked `aria-disabled` when expanded in single select mode since the button is no longer interact-able. This is outlined in the aria spec.

```
If the accordion panel associated with an accordion header is visible, and if the accordion does not permit the panel to be collapsed, the header button element has aria-disabled set to true.
```
